src/core/core.scale.js

```diff
@@ -7,6 +7,8 @@ var Ticks = require('./core.ticks');
 
 var valueOrDefault = helpers.valueOrDefault;
 var valueAtIndexOrDefault = helpers.valueAtIndexOrDefault;
+var parseFont = helpers.options._parseFont;
+var toPadding = helpers.options.toPadding;
```

Comment:
This would reduce the size:
```js
var optionsHelpers = helpers.options;
var parseFont = optionsHelpers._parseFont;
var toPadding = optionsHelpers.toPadding;

// ...

		color: optionsHelpers.resolve([nestedOpts.fontColor, options.fontColor, defaults.global.defaultFontColor])
```

Reply:
done